pkg/services/gitops/install.go

```diff
@@ -32,9 +33,22 @@ func (g *Gitops) Install(params InstallParams) ([]byte, error) {
 	if params.DryRun {
 		fluxManifests = append(fluxManifests, manifests.AppCRD...)
 	} else {
+		ctx := context.Background()
+		if err := g.kube.CreateSecret(ctx, kube.GitTokenSecretName, kube.GitTokenKeyName, params.GitToken, params.Namespace); err != nil {
+			return nil, errors.Wrap(err, "could not create git token secret")
+		}
 		if out, err := g.kube.Apply(manifests.AppCRD, params.Namespace); err != nil {
 			return []byte{}, errors.Wrapf(err, "failed to apply App spec CR: %s", string(out))
 		}
+		if out, err := g.kube.Apply(manifests.ServiceAccountApiService, params.Namespace); err != nil {
+			return []byte{}, errors.Wrapf(err, "failed to apply service account manifest for api-service: %s", string(out))
```

Comment:
after some discussion with Kevin we've decided to move to `fmt.Errorf`. You don't need to change all the codebase, but as we work through the packages we should take the chance to change the calls there.

You can see more about it: https://blog.golang.org/go1.13-errors